udf/coalesce_adjacent_days_bytes.sql

```diff
@@ -0,0 +1,28 @@
+/*
+Coalesce previous data's PCD with the new data's PCD.
+
+We generally want to believe only the first reasonable profile creation
+date that we receive from a client.
+Given bytes representing usage from the previous day and the current day,
+this function shifts the first argument by one day and returns either that
+value if non-zero and non-null, the current day value if non-zero and non-null,
+or else 0.
+*/
+
+
+
+CREATE TEMP FUNCTION
+  udf_coalesce_adjacent_days_bytes(prev BYTES, curr BYTES) AS (
```

Comment:
these functions are all named with `bytes`, presumably to match the existing `bits` functions that store 28 days of history. I think don't think this naming is sufficiently descriptive to avoid confusing users who access our library of persistent udfs.

I propose that we rename the existing functions to be named `28bits` and these functions to be named `365bits`.

cc @jklukas

Reply:
```suggestion
  udf_coalesce_adjacent_days_365bits(prev BYTES, curr BYTES) AS (
```

Reply:
I agree that it makes sense to specify the number of bits in the function names, though I dislike the lack of underscore in `28bits`. We do already use `mau28` in the `exact_mau28` table names. Somehow, the number as suffix seems more palatable than as prefix.

So I'd prefer one of:

```
udf_coalesce_adjacent_days_bits365
udf_coalesce_adjacent_days_365_bits
```

Reply:
My only other caveat is it's different types for the two. This is using `bytes`, while the existing one use `int`. However I doubt they'll get used at all outside our queries so perhaps it doesn't matter.

Reply:
I also prefer `*_N_bits`, so let's go with that.